### PR TITLE
Hey Peter what do you think of that

### PR DESCRIPTION
**Pull request by dooshbag14**
i made an additional module

### CODE REVIEW

File: first [hunk 1]
```diff
@@ -25,12 +25,16 @@ PIETROOOOOO BADASS OO777777777==111111==7777777777777777
...
 
-
 <mod1>
-d14 content here
+ d13 content goes here...
 </mod1>
 
 
+<mod2>
+d14 content here
```
Comment (dooshbag13) [inline first]: its good here

File: first [hunk 2]
```diff
@@ -25,12 +25,16 @@ PIETROOOOOO BADASS OO777777777==111111==7777777777777777
...
+</mod2>
+
+
```
Comment (dooshbag13) [inline first]: remove additional spaces here

File: first [hunk 3]
```diff
@@ -25,12 +25,16 @@ PIETROOOOOO BADASS OO777777777==111111==7777777777777777
 hahaha1==GGGGGGGGGGGG1
```
Comment (dooshbag13) [inline first]: less space here also

**Comment by dooshbag13** [comment]
great job mate

**Comment by dooshbag14** [comment]
i did it all

**Comment by dooshbag13** [comment]
great job thanks
merging now..

**Pull request merged**